cg/models/scout/validators.py

```diff
@@ -0,0 +1,3 @@
+def field_not_none(value: str):
+    assert value is not None
```

Comment:
We should avoid asserting in production code. Pydantic also warns about this.

Reply:
Also - can we just not avoid setting a default value for the field it validates?

Reply:
That is very true, I'll remove the validator. It does not make sense

Reply:
Well, removing the validator caused a bunch of tests to break. Not opening that can of worms today. Replaced the assert.

Reply:
The issue is that other models inherit from the `ScoutIndividual` class. It seems like the None validation is only run when `ScoutIndividual` is instantiated directly. So if for instance `sample_id` is marked as required, it suddenly becomes required in all the models inheriting from the base class.

I think this inheritance pattern is not so good here. It causes confusion and weird behaviour when refactoring. If data is not needed in subclasses, the subclasses has no business inheriting from the class in the first place